Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Extensions/FormExtension.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Aquality.Selenium.Forms;
+using NUnit.Framework;
+
+namespace Aquality.Selenium.Template.NUnit.Extensions
+{
+    public static class FormExtensions
+    {
+        public static void AssertIsPresent(this Form form)
+        {
+            Assert.IsTrue(form.State.WaitForDisplayed(), $"{form.Name} should be presented");
+        }
+
+        public static void AssertIsNotPresent(this Form form)
+        {
+            Assert.IsTrue(form.State.WaitForNotDisplayed(), $"{form.Name} should be presented");
```

Comment:
The message is wrong, must be "should not be present"

Reply:
This code was updated.